plugins/templates-maven-plugin/src/main/java/com/google/cloud/teleport/plugin/maven/TemplatesStageMojo.java

```diff
@@ -699,10 +710,20 @@ private void stageFlexPythonTemplate(
       String containerName,
       String templatePath)
       throws IOException, InterruptedException, TemplateException {
-
-    String dockerfilePath = outputClassesDirectory.getPath() + "/" + containerName + "/Dockerfile";
+    String dockerfileContainer = outputClassesDirectory.getPath() + "/" + containerName;
+    String dockerfilePath = dockerfileContainer + "/Dockerfile";
+    LOG.info("Generating dockerfile " + dockerfilePath);
     File dockerfile = new File(dockerfilePath);
     if (!dockerfile.exists()) {
+      // Copy in requirements.txt if present
+      File sourceRequirements = new File(outputClassesDirectory.getPath() + "/requirements.txt");
+      File destRequirements = new File(dockerfileContainer + "/requirements.txt");
+      if (sourceRequirements.exists()) {
+        Files.copy(
+            sourceRequirements.toPath(),
+            destRequirements.toPath(),
+            StandardCopyOption.REPLACE_EXISTING);
+      }
```

Comment:
nit: Why is this needed?

Cloud Build runs in the `outputClassesDirectory`, so it shouldn't even be looking at the `dockerfilePath`

Reply:
`dockerfilePath` is in `outputClassesDirectory`, so these are related. I found that as it stands right now (without my change), the requirements.txt files were not being found even when present, but colocating them with the dockerfile allowed docker to find them without issue.

Reply:
But it definitely does fail without this change

Reply:
Ok interesting because we used to use requirements files and they were stored in the resources directory of the template, so not sure what changed. But ok SGTM